README.md

```diff
@@ -22,8 +22,10 @@ WSTest is the main test running class in pywsitest. It currently has the followi
 - **with_message**: add a message for the test runner to send on connection
 - **with_response_timeout**: set the timeout in seconds for the test runner to wait for a response from the websocket
 - **with_message_timeout**: set the timeout in seconds for the test runner to wait while trying to send a message to the websocket
+- **with_request_timeout**: set the timeout in seconds for the rest request attached to the instance of this class
 - **with_test_timeout**: set the timeout in seconds for the test runner to run for
 - **with_received_response_logging**: enable logging of received responses on response timeout error
+- **with_request**: attach a rest api request to the instance of this class
```

Comment:
Can this be moved so it's next to `with_response` and `with_message`?
I'd prefer if it were grouped that way